northcoders/frontend/games-project/src/components/CommentAdder.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from "react";
+
+const CommentAdder = () =>{
+    const [newComment, setNewComment] = useState('');
+    const handleSubmit = (e) =>{
+        e.preventDefault();
+        // postComment(newComment)
+        // .then
```

Comment:
I can't actually add a comment, I suspect that as there's no code here to do the actual api call would be the culprit